$MirOS: ports/mailnews/alpine/patches/patch-imap_src_c-client_mail_h,v 1.1 2008/05/07 22:28:40 tg Exp $
--- imap/src/c-client/mail.h.orig	2008-08-08 19:34:22.000000000 +0200
+++ imap/src/c-client/mail.h	2009-02-02 20:58:00.000000000 +0100
@@ -26,7 +26,7 @@
 
 /* The Version */
 
-#define CCLIENTVERSION "2007c"
+#define CCLIENTVERSION "2007cL"
 
 /* Build parameters */
 
